src/rtc/rtc_s1_single_job.py

```diff
@@ -1170,6 +1277,11 @@ def run_single_job(cfg: RunConfig):
             logger.info(f'    reading burst SLCs')
 
         radar_grid = burst.as_isce3_radargrid()
+        if processing_type == 'STATIC_LAYERS':
+            radar_grid = radar_grid.offset_and_resize(- int(1.5 * radar_grid.length), 
+                                                      - int(radar_grid.width),
+                                                      int(4 * radar_grid.length),
+                                                      int(3 * radar_grid.width))
```

Comment:
I think this is where you adjusted the radargrid for static layers that you mentioned. I have a couple of questions
- It the adjustment to make sure that the static layers from a point of time covers the RTC image area in different times?
- Can you explain me about the parameters in `radargrid.offset_and_resize`?

Reply:
Hi @seongsujeong , that's right. This is the point that we increase the radargrid to cover the burst grid. `radargrid.offset_and_resize()` returns a new radar grid object based on a reference radar grid. The first two parameters are the offset, in pixels, of the start of the new grid. The last two parameters are the dimensions of the new grid. As you can see, we are starting 1.5 times in length and 1 time in width earlier, and we have the same for the end positions. We have these values because radar grid is shorter in length than it's wide.  

For example, if the radar grid has dimensions 100 by 100 starting at (y0, x0) and ending at (yf, xf). We'll start the new grid at the position (y0 - 150, x0 - 100), and it will have dimensions of (length: 400, width: 300), which means that the end position will be (y0 + 250, x0 + 200) which is equal to (yf + 150, xf + 100).

Since radar grid is shorter in length than it's wide, I add

Reply:
Yes, the time to process will be much slower. I'm working on updating ISCE3 topo to run faster. Please note that this longer process time applies only to static layers, which will be generated only once or a limited number of times.

The region added to the radargrid is symmetric. In the example above, we are adding (-150,-100) to the starting point and  (150, 100) to the ending point.